cmd/verifydependencies/verifydependencies.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// verify that dependencies are up-to-date across different files
+package main
+
+import (
+	"flag"
+	"log"
+	"strings"
+
+	"bufio"
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"gopkg.in/yaml.v2"
+)
+
+type dependencies struct {
```

Comment:
late to the PR, this is a good approach to get kubernetes dependencies.
is it possible to expose types somewhere?

Reply:
I'm hoping this approach is temporary. This needs more though before we expose it as an API

Reply:
@figo - I agree with @liggitt I wouldn't expose this as an API, users can still consume this file, If we want to do so, we'd need to think this through

Reply:
that is a good step to take at some point - machine readable dep list.